bin/run.cmd

```diff
@@ -1,3 +1,36 @@
 @echo off
+setlocal enableextensions
 
-node "%~dp0\run" %*
+set SFDX_INSTALLER=false&set BIN_NAME=run
```

Comment:
Is adding all of this necessary?

Reply:
For now, I will revert the file's content